# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1461270954 14400
#      Thu Apr 21 16:35:54 2016 -0400
# Node ID 3d99ba3d08c1545ad15d3eed4b9bccc0327eee55
# Parent  59ebef9680eff6bc1a8893fe9064d53ef97a540e
# Parent  cff44a19abfecebf8d3e181dcaf060ca445a1bc4
maint: merge stable to default.

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -95,8 +95,9 @@ eba80000fa0dad32ba0f1cd767dd826d1ce1aba6
 065f933ef08318e40b81f7fe75236e6175088117 rc-4-0-0-3
 42bb3a776c9fcc008669f382d2409297c4a901b3 rc-4-0-0-4
 3ccc2d02e64b61c6827009d42e5b12274e7ba5fc release-4-0-0
 20b17dda0e0c838646dfdccec153ec8cdf6d3c37 rc-4-0-1-1
 0cfcc178432bfc7cfdef5dca96e74ec5884ba752 rc-4-0-1-2
 a58d5e6df908199c4aea83f78077b81289d2a0ad rc-4-0-1-3
 2b66f30a0de7a09801c2a3566021cde7e194d3d3 rc-4-0-1-4
 5162cdfb56d8e31ad00dc260a26d1c92c69c2acf release-4-0-1
+acaf1e137c5e6f201a6689a99a6568d0dfe9e9ed release-4-0-2
